packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+
+- We add the new data source caption to the typings generator.
```

Comment:
This needs to be more specific, as the caption of a data source is part of the _preview_ typings. We also make it a habit to explicitly mention the Mendix version in which a feature was added (which is usually the same as the version of the tools, but not necessarily).

Reply:
@hendrik-jacobs-mendix Can you also bump the version in the package.json to 9.24.0, as promised in the PR description? ;)

Reply:
✔️